build/config/babel/bundler.js

```diff
@@ -3,8 +3,8 @@
 var Path = require('path');
 var fs = require('fs');
 
-var moduleIds = new Set();
-var modules = [];
+var moduleIds;
+var modules;
 
 /*
     const modules = new Map();
```

Comment:
I think I misread the babel code below.  What does `t.newExpression(t.identifier('Map'), []))` do?  In particular, the `[]` at the end?